tools/frb_internal/lib/src/makefile_dart/lint.dart

```diff
@@ -54,18 +55,21 @@ Future<void> lintRustFormat(LintConfig config) async {
 
 Future<void> lintRustClippy(LintConfig config) async {
   for (final package in kRustPackages) {
+    String? feature = getRustFeaturesOfPackage(package);
     if (config.fix) {
-      await exec('cargo fix --allow-dirty --allow-staged',
+      await exec(
+          'cargo fix ${feature != null ? "--features $feature" : ""} --allow-dirty --allow-staged',
           relativePwd: package);
     }
     await exec(
-        'cargo clippy ${config.fix ? "--fix --allow-dirty --allow-staged" : ""} -- -D warnings',
+        'cargo clippy ${feature != null ? "--features $feature" : ""} ${config.fix ? "--fix --allow-dirty --allow-staged" : ""} -- -D warnings',
         relativePwd: package);
   }
 
   for (final package in kRustPackagesAllowWeb) {
+    String? feature = getRustFeaturesOfPackage(package);
```

Comment:
```suggestion
    final feature = getRustFeaturesOfPackage(package);
```

(you can use github and click the button to quickly accept this tiny change)